share/tools/web_config/sample_prompts/sorin.fish

```diff
@@ -0,0 +1,60 @@
+# Name:   Sorin
+# Author: Ivan Tham <ivanthamjunhoe@gmail.com>
+
+# If login shell, don't do anything
+if status --is-login; and function fish_prompt; echo -n '% '; end; end
+
+function fish_prompt
+	if not set -q -g __prompt_sorin_functions-defined
+		set -g __prompt_sorin_functions-defined
+	end
+
+	test $SSH_TTY; and printf (set_color red)(whoami)(set_color white)'@'(set_color yellow)(hostname)' '
```

Comment:
I want that **error** returned for **`and`** to execute the code when it is *true*.